lib/linguist/languages.yml

```diff
@@ -3968,6 +3968,14 @@ PowerShell:
   interpreters:
   - pwsh
   language_id: 293
+Prisma:
+  type: data
+  color: "#162d3b"
```

Comment:
The colour is only used in the statistics-bar, which shows the usages of `programming` and `markup`-type languages only. So the `color` field needs to be removed (also the reason the tests are failing).

Reply:
Won't this language show up in a grey color if `linguist-detectable` is used?